Plugins/iOS/upload-dsym.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
```

Comment:
Unity generates an Xcode project. I didnt try adding the upload-dsym script, but i'd imagine that If i have an xcode project, i can add the run script.